docs/using-openstack-cloud-controller-manager.md

```diff
@@ -33,7 +33,7 @@ The following guide has been tested to install Kubernetes v1.17 on Ubuntu 18.04.
 
 ### Steps
 
-- Create the kubeadm config file according to [`manifests/controller-manager/kubeadm.conf`](https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/kubeadm.conf)
+- Create the kubeadm config file according to [`manifests/controller-manager/kubeadm.conf`](https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/kubeadm.conf), you need modify the kubeadm version configuration according to your settings.
```

Comment:
The content you added is still not very clear to readers, what's the motivation of the addition?

Reply:
I actually copied the contents but 
the version in that file doesn't fit for my k8s version
just a reminder for someone to try this, they need change the version accordindly